tests/data/test_loader.py

```diff
@@ -46,38 +39,91 @@ def test_load_general_data(facade, monkeypatch):
 
     facade.load_gen_data.assert_called_once_with("test_case", "some_key", 1)
 
-    assert result.equals(create_expected_data())
+    assert result.equals(
+        pd.DataFrame(
+            [[10.0, 10.0, 10.0, 10.0]],
+            index=[0],
+        )
+    )
 
 
-@pytest.mark.usefixtures("facade")
 def test_load_general_obs(facade, monkeypatch):
     mock_node = MagicMock()
     mock_node.__len__.return_value = 3
-    mock_node.get_data_points.return_value = [10.0, 10.0, 10.0]
-    mock_node.get_std.return_value = [1.0, 1.0, 1.0]
+    mock_node.get_data_points.return_value = [10.0, 20.0, 30.0]
+    mock_node.get_std.return_value = [1.0, 2.0, 3.0]
     mock_node.getIndex.side_effect = mocked_obs_node_get_index_nr
 
-    facade.load_gen_data.return_value = pd.DataFrame(data=[10.0, 10.0, 10.0, 10.0])
+    obs_mock = Mock()
+    obs_mock.getDataKey.return_value = "test_data_key"
+    obs_mock.getStepList.return_value.asList.return_value = [1]
+    facade.get_observations.return_value = {"some_key": obs_mock}
+
+    facade.load_gen_data.return_value = pd.DataFrame(data=[9.9, 19.9, 29.9, 39.9])
     facade.get_observations()["some_key"].getNode.return_value = mock_node
 
-    result = loader._load_general_obs(facade, ["some_key"], "test_case")
+    result = loader._load_general_obs(facade, ["some_key"], "a_random_name")
 
-    mock_node.get_data_points.assert_called_once()
-    mock_node.get_std.assert_called_once()
+    mock_node.get_data_points.assert_called_once_with()
```

Comment:
Out of curiosity, why change to `assert_called_once_with()`? I would've understood there was something passed as an argument..

Reply:
Because it is more explicit, as `assert_called_once` has less information carried with it.

Reply:
So this has more to do with developer feedback when tests fail than anything else?

Reply:
Yes, that was the intention.